app/docker/views/containers/console/exec.html

```diff
@@ -1,17 +1,17 @@
 <rd-header>
   <rd-header-title title-text="Container console"></rd-header-title>
-  <rd-header-content ng-if="state.loaded">
+  <rd-header-content>
     <a ui-sref="docker.containers">Containers</a> &gt; <a ui-sref="docker.containers.container({id: container.Id})">{{ container.Name|trimcontainername }}</a> &gt; Console
   </rd-header-content>
 </rd-header>
 
-<div class="row" ng-if="state.loaded">
+<div class="row" ng-init="initView()" ng-show="state > states.unloaded">
   <div class="col-lg-12 col-md-12 col-xs-12">
     <rd-widget>
-      <rd-widget-header icon="fa-terminal" title-text="Console"></rd-widget-header>
+      <rd-widget-header icon="fa-terminal" title-text="Execute"></rd-widget-header>
       <rd-widget-body>
         <form class="form-horizontal">
-          <div ng-if="!state.connected">
+          <div ng-if="state < states.connecting">
```

Comment:
Not easy to read without knowing the states and how they are ordered.
I'd prefer using ng-if="state === states.disconnected" here.

Reply:
Done